README.md

```diff
@@ -127,7 +127,7 @@ Method of smart contract deployment:
 // Deploy contract in sync and async way.
 public AppSendTransaction deploy(File contractFile, String nonce, BigInteger quota)
 
-public CompletableFuture<AppSendTransaction> deployAsync(File contractFile, String nonce, BigInteger quota)
+public Future<EthSendTransaction> deployAsync(File contractFile, BigInteger nonce, BigInteger quota)
```

Comment:
This should be `Future<AppSendTransaction>` rather than `Future<EthSendTransaction>`